client/modules/User/components/DashboardTabSwitcher.jsx

```diff
@@ -57,6 +79,51 @@ const DashboardTabSwitcher = ({ currentTab, isOwner, username }) => {
           </Tab>
         )}
       </div>
+      <MediaQuery maxWidth={770}>
+        {(mobile) => (
+          <FilterOptions className={mobile ? 'visible' : 'hidden'}>
+            <div>
+              <IconButton icon={FilterIcon} />
+              <ul>
+                <li>
+                  <button
+                    onClick={() => dispatch(toggleDirectionForField('name'))}
+                  >
+                    Name
```

Comment:
We have translation keys that we can use here.  There's a different set of keys for `SketchList` and `CollectionList` but the values are the same so whatever.  Can use `t('CollectionList.HeaderCreatedAt')` etc.